mbtiles/src/mbtiles.rs

```diff
@@ -39,6 +39,60 @@ pub struct Metadata {
     pub json: Option<JSONValue>,
 }
 
+#[derive(Clone, Debug, PartialEq, Serialize)]
+pub struct LevelDetail {
+    pub zoom: String,
+    pub count: u32,
+    pub smallest: f32,
+    pub largest: f32,
+    pub average: f32,
+    pub bounding_box: [f32; 4],
```

Comment:
```suggestion
    pub bounds: tilejson::Bounds,
```

You may want to add a `use tilejson::Bounds` at the top instead.